lib/kitchen/verifier/inspec.rb

```diff
@@ -214,6 +214,12 @@ def runner_options(transport, state = {}, platform = nil, suite = nil) # rubocop
             # default to false until we default to true in inspec
             runner_options[:backend_cache] = false
           end
+          
+          if config[:backend]
+            runner_options[:backend] = config[:backend]
+          else
+            runner_options[:backend] = 'ssh'
```

Comment:
Why are we setting this as default?

Reply:
I think that's the current default isn't it?

Reply:
I think explicitly setting it here is surprising, however. If config[:backend] is unset, let's just leave runner_options[:backend] unset, and it will determine its own default.

Reply:
Please see previous comment